tests/Bridge/Symfony/Validator/Metadata/Property/ValidatorPropertyMetadataFactoryTest.php

```diff
@@ -402,7 +402,7 @@ public function testCreateWithAtLeastOneOfConstraint(): void
         $this->assertNotNull($schema);
         $this->assertArrayHasKey('oneOf', $schema);
         $this->assertSame([
-            ['pattern' => '.*#.*'],
+            ['pattern' => '^(?:.*#.*)$'],
```

Comment:
note: it would be nicer to get `'#'` here, but in general the wrapped part could be anything...